core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -304,6 +316,42 @@ class WorkspaceService(protected val userInfo: UserInfo,
     })
   }
 
+  private def getAzureCloudContextFromWorkspaceManager(workspaceContext: Workspace, parentSpan: Span = null) = {
+    workspaceContext.workspaceType match {
+      case WorkspaceType.McWorkspace => {
+        val span = startSpanWithParent("getWorkspaceFromWorkspaceManager", parentSpan)
+
+        try {
+          val wsmInfo = workspaceManagerDAO.getWorkspace(workspaceContext.workspaceIdAsUUID, userInfo.accessToken)
+
+          Option(wsmInfo.getAzureContext) match {
```

Comment:
What would it mean if the Azure context is not present? I'm wondering if we should do some error handling/logging in that case.

Reply:
It'd likely mean it's a gcp workspace. At this point in the code's evolution agreed this is an error case, I'll add some handling.